Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/FeatureClassUtils.cs

```diff
@@ -311,7 +313,7 @@ List<ObjectConversionTracker> listOfTrackers
       {
         if (trackerItem.HostAppGeom is not ACG.Geometry shape)
         {
-          throw new SpeckleConversionException("Feature Class element had no converted geometry");
+          throw new ValidationException("Feature Class element had no converted geometry");
```

Comment:
possibly we can replace some of the ArgumentExceptions in other converters to ValidationException too